docs/getting-started/usage/layout.md

```diff
@@ -0,0 +1,47 @@
+# Layout
```

Comment:
Also maybe consider making this page a bit shorter, it has a lot of text which is all great, but I'm afraid people will just be *tl;dr* Nothing major, just maybe merging some paragraphs or some redundant sentences. @philsturgeon ?